_sass/theme/_base.scss

```
@@ -1,12 +1,4 @@
 /* ======= Base ======= */
-@font-face {
-    font-family: 'tuos_stephensonbold';
-    src: url('/assets/fonts/stephbd-webfont.woff2') format('woff2'),
-    url('/assets/fonts/stephbd-webfont.woff') format('woff');
-    font-weight: normal;
-    font-style: normal;
-
-}
 
 body {
     font-family: 'Lato', arial, sans-serif;   
```

If you declare the new fonts `uos-sans` and `uos-serif` (we might even want to add our own `uos-mono` which is source code pro, given its the same family but the brand guidelines don't speficy a mono font), then they can be used within this file for the theme's `font-family` specifications. Its worth maintaining backup fonts on the offchance someone views the site with a browser which cannot open the custom font files for whateve reason. 

This would need repeating for more occurances than just this one, as the theme has more specific selectors that use different fonts unfortunately.

```suggestion
    font-family: 'uos-sans', arial, sans-serif;   
Write
```